waspc/examples/todoApp/README.md

```diff
@@ -0,0 +1,58 @@
+# TodoApp
+
+The main purpose of this app is for contributors to easily test the version of the `wasp` CLI that they are working on right now and therefore aid them during development.
+
+That is why it is a bit crowded with features and somewhat incoherent: it is not an app to be used as a showcase, but an app that we use as a "workbench" / "playground" while working on `wasp`.
+
+This app should always be "up to date", in the sense that we keep updating it to always work with the latest changes to `wasp`. This means you can count on it being up to date, but also that you should make sure to keep it up to date as you do changes. You should also be adding new features to it if you implement them, so they can easily be tested / tried out.
+
+## Set up
+
+### Env vars
+
+To do the minimum to get the app running, do:
+```sh
+cp .env.server.example .env.server
+```
+This will create a `.env.server` file with some basic env vars set to dummy values. This will allow your app to run and work, but not all features will work, e.g. Google Auth won't work because it needs you to provide real API keys via env vars.
+
+To obtain all the API keys to get all the features of the app working, if you do need that, you will have to obtain them on your own, normally from the corresponding third-party services. Check Wasp docs for instructions on how to do this for Google Auth, GitHub Auth, and so on.
```

Comment:
I'd skip this half step and just insist on pulling the dotenv thing.

Btw, can we make this easier for non-team contributors. I know we don't have any, but still :smile:

Reply:
I am not sure what you mean by "half-step" -> this is a step you need to do if you are extenral contributor. If you are internal contributor, you can use the dotenv, as explained below.

I would love to make it easier for external contributors, but I don't know how! But I think should be fine for now -> it is not suepr easy for them, but they can get the app running even without it thanks to .env.server.example, and they know exactly what they need to do to go further. On the other hand, for internal contributors it is easy and they will be needing this the most.

Reply:
Yeah, forgot about the role this "half-step" might have for external contributors - all good.